rust/datafusion/src/execution/physical_plan/hash_aggregate.rs

```diff
@@ -111,6 +123,17 @@ impl ExecutionPlan for HashAggregateExec {
         self.schema.clone()
     }
 
+    fn children(&self) -> Vec<Arc<dyn ExecutionPlan>> {
+        vec![self.input.clone()]
+    }
+
+    fn required_child_distribution(&self) -> Distribution {
+        match &self.mode {
+            AggregateMode::Partial => Distribution::UnspecifiedDistribution,
```

Comment:
I would think the output distribution is likely the same as the input distribution if the input distribution is on the grouping keys. THe only possibly change here is if the mode is `Partial` and the input distribution is `SinglePartition` so is the output